pkg/queue/protobuf_stats_reporter_test.go

```diff
@@ -51,22 +51,14 @@ func TestReporterReport(t *testing.T) {
 				RequestCount:              test.reqCount,
 				ProxiedRequestCount:       test.proxiedReqCount,
 			})
-			stat := reporter.stat.Load().(metrics.Stat)
-			if stat.RequestCount != test.expectedReqCount {
-				t.Errorf("stat.RequestCount = %v, want %v", stat.RequestCount, test.expectedReqCount)
-			}
-			if stat.AverageConcurrentRequests != test.expectedConcurrency {
-				t.Errorf("stat.AverageConcurrentRequests = %v, want %v", stat.AverageConcurrentRequests, test.expectedConcurrency)
-			}
-			if stat.ProxiedRequestCount != test.expectedProxiedRequestCount {
-				t.Errorf("stat.ProxiedRequestCount = %v, want %v", stat.ProxiedRequestCount, test.expectedProxiedRequestCount)
-			}
-			if stat.AverageProxiedConcurrentRequests != test.expectedProxiedConcurrency {
-				t.Errorf("stat.AverageProxiedConcurrentRequests = %v, want %v", stat.AverageProxiedConcurrentRequests, test.expectedProxiedConcurrency)
-			}
-			if got := stat.ProcessUptime; got < 5.0 || got > 6.0 {
-				t.Errorf("Got %v for process uptime, wanted 5.0 <= x < 6.0", got)
```

Comment:
This check is lost with the new method I believe? We should keep it alive as a separate check then.